opentelemetry-proto/src/proto/tonic/opentelemetry.proto.common.v1.rs

```diff
@@ -68,18 +72,28 @@ pub struct KeyValueList {
 /// attributes, etc.
 #[cfg_attr(feature = "with-schemars", derive(schemars::JsonSchema))]
 #[cfg_attr(feature = "with-serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "with-serde", serde(rename_all = "camelCase"))]
 #[allow(clippy::derive_partial_eq_without_eq)]
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct KeyValue {
     #[prost(string, tag = "1")]
     pub key: ::prost::alloc::string::String,
     #[prost(message, optional, tag = "2")]
+    #[cfg_attr(
```

Comment:
May I ask why this block was added? It is inconsistent with what the serde semantics are as implemented on the (Golang-based) OTEL Collector - [see this example](https://github.com/open-telemetry/opentelemetry-collector/blob/main/pdata/ptrace/json_test.go#L91) 

In my local fork I've removed this and only then am I able to successfully unmarshal from within the Collector.